src/MultiRegion/cubed_sphere_boundary_conditions.jl

```diff
@@ -0,0 +1,392 @@
+function find_neighboring_panels(region)
+    if mod(region, 2) == 1
+        region_E = mod(region + 0, 6) + 1
+        region_N = mod(region + 1, 6) + 1
+        region_W = mod(region + 3, 6) + 1
+        region_S = mod(region + 4, 6) + 1
+    elseif mod(region, 2) == 0
+        region_E = mod(region + 1, 6) + 1
+        region_N = mod(region + 0, 6) + 1
+        region_W = mod(region + 4, 6) + 1
+        region_S = mod(region + 3, 6) + 1
+    end
+
+    return (; region_E, region_N, region_W, region_S)
+end
+
+function fill_cubed_sphere_halo_regions!(field::CubedSphereField{<:Center, <:Center})
```

Comment:
why we use new method name `fill_cubed_sphere_halo_regions!` here and we don't simply add a method to the `fill_halo_regions!(::CubedSphereField)`

?

Reply:
Taken care of in commit [c8cf9bb](https://github.com/CliMA/Oceananigans.jl/pull/3488/commits/c8cf9bb06eecca51749771093cc971c5dd2b1b37).

Reply:
done